### ISSUE TITLE
[BUG] High refresh rate (>60 Hz) not effectively applied on multi-monitor setup

### ISSUE THREAD

**Issue by Crystal4276** [linuxmint/cinnamon#11133]
```
 * Cinnamon version: Cinnamon 5.4.9
 * Distribution: Arch
 * Graphics hardware *and* driver used:
     -- RTX 2080
     -- nvidia-open, driver Version: 515.65.01 
 * 64 bit
 ```
### Issue
High refresh rate (>60 Hz) not effectively applied on multi-monitor setup when others monitors are running at 60Hz

### Steps to reproduce

I have three monitors, one main able to run at 100Hz and two old monitors that have a max refresh rate of 60Hz.
Configuration in display utility with 100Hz for the main monitor and 60 Hz for the two other monitors results in 60 Hz for all (although display utility keep showing 100 Hz for main monitor).
Physically disconnecting the two 60Hz monitors from the GPU, finally enable the 100Hz refresh rate on main monitor.

### Expected behaviour

Setting the refresh rate of monitors according to their respective specification in display utility should work.

**Comment by SpinningVinyl** [comment]
I believe this is Nvidia's fault, check this out: https://www.reddit.com/r/linuxhardware/comments/mht7kn/workaround_for_multiple_monitors_with_different/

**Comment by Crystal4276** [comment]
Hi,
Thanks for link ! 
I'm a bit afraid of about display utility from cinnamon and nvidia-setting interacts. 
But worth a try...

Doing my own googling of the issue, it seems that the problem disappears when using Wayland instead of X
Although I don't understand the issue, it doesn't seem being entirely on Nvidia shoulders.

**Comment by SpinningVinyl** [comment]
Yes, Xorg is notoriously bad with per-monitor configurations because it basically treats all monitors as one desktop split into parts instead of separate devices. Wayland is much better in this regard, but Cinnamon does not run on Wayland.

**Comment by pablosme** [comment]
Kinda related to your problem, I was getting insane because my refresh rate was messed up (I have 3 60hz monitors, 2x24' and 1x32' 4k) but couldnt find the reason. After checking out your problem I realized one of my secundary monitors were 50hz for some reason. Probably altered the config without realizing when using the mouse scroll
